<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="org.springframework.cache.interceptor.CacheSyncFailureTests" tests="5" skipped="0" failures="0" errors="0" timestamp="2021-05-01T09:32:14" hostname="wangweilideMBP.lan" time="0.048">
  <properties/>
  <testcase name="syncWithTwoGetOperations()" classname="org.springframework.cache.interceptor.CacheSyncFailureTests" time="0.015"/>
  <testcase name="unlessSync()" classname="org.springframework.cache.interceptor.CacheSyncFailureTests" time="0.007"/>
  <testcase name="severalCachesWithResolvedSync()" classname="org.springframework.cache.interceptor.CacheSyncFailureTests" time="0.009"/>
  <testcase name="severalCachesSync()" classname="org.springframework.cache.interceptor.CacheSyncFailureTests" time="0.008"/>
  <testcase name="syncWithAnotherOperation()" classname="org.springframework.cache.interceptor.CacheSyncFailureTests" time="0.008"/>
  <system-out><![CDATA[]]></system-out>
  <system-err><![CDATA[]]></system-err>
</testsuite>
